Apps/Sandcastle/gallery/CZML Different Coordinate Maps.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">  <!-- Use Chrome Frame in IE -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="CZML Point">
```

Comment:
The description is out of sync with the title.

Also, `CZML Different Coordinate Maps` should be something like `CZML Position Definitions`.